frontend/packages/dev-console/src/components/import/CreateSecretModal.tsx

```diff
@@ -42,8 +44,8 @@ const CreateSecretModal: React.FC<Props> = ({ close, namespace, save, secretType
       onSave={handleSave}
       fixed={{ metadata: { namespace } }}
       secretTypeAbstraction={secretType}
-      titleVerb="Create"
       isCreate
+      t={t}
```

Comment:
You shouldn't have to pass in `t`. The component should handle it themselves using the hook or the hoc.